hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -254,6 +254,12 @@
       .withDocumentation("The average record size. If not explicitly specified, hudi will compute the "
           + "record size estimate compute dynamically based on commit metadata. "
           + " This is critical in computing the insert parallelism and bin-packing inserts into small files.");
+  
+  public static final ConfigProperty<Boolean> ALLOW_MULTIPLE_CLEANS = ConfigProperty
+      .key("hoodie.allow.multiple.cleans")
```

Comment:
nit: rename to `hoodie.clean.allow.multiple` or `hoodie.cleaner.allow.multiple`?  Also, shouldn't all clean configs be in new `HoodieCleanConfig` class (can be a separate PR)?

Reply:
In general, its a good practice to name the variable as you read it. for eg, the getter method of this variable reads as allowMultipleCleans() which goes in line w/ the config key. 

If you check few other existing configs, it does align well. but I want to fix that going forward. for eg:hoodie.clean.automatic , variable is called AUTO_CLEAN and method is called isAutoClean. I would have preferred the config key to be "hoodie.auto.clean".

Reply:
Got it.  I come from the perspective that all the configs for CLEAN action should have the same prefix, so that it's easier for the user to discover relevant knobs.  The suffix can be more descriptive.  I also think that the description of the config should be more readable than only making config naming readable.

Reply:
sounds good, will fix.